### DISCUSSION TITLE
Appstore pricing

### DISCUSSION THREAD

**Discussion by gabeklavans** [cooklang/cooklang-ios-app#20] [General]
I'm by no means complaining, but I did a double-take when I saw the $12 price tag of the app on the appstore... It feels a bit out of place for this free and open standard to have an official app that charges an unusually large amount for an iOS app.

I'm happy to donate to the cause, but I'm reluctant to even try this app at that price, and I'm sure it puts off others way more. Any issues they run into are going to create much more frustration knowing how much they paid for the app when the vast majority of apps don't even go past $3.

Could lowering the price be a consideration?

**Comment by ringods** [comment]
I'm not the author, but I think $12 is a fair price if you want the author to continue investing time in it. I immediately bought it when I found out there was an app for CookLang.

A race to the bottom, in this case regarding the pricing of apps, usually doesn't end well, neither for the author and the users.

If I would complain on one thing, then I would complain on Apple taking a 30% margin just for hosting the app in the AppStore. Luckily, the European Union is providing push back on this monopolistic situation.
